src/cookbook/games/achievements-leaderboard.md

```diff
@@ -0,0 +1,352 @@
+---
+title: Add achievements and leaderboards to your mobile game
+description: >
+  How to use the games_services plugin to add functionality to your game.
+---
+
+Gamers have various motivations for playing games. In broad strokes,
+there are four major motivations: 
+[immersion, achievement, cooperation, and competition](https://meditations.metavert.io/p/game-player-motivations).
+No matter the game you build, some players want to *achieve* in it. This
+could be trophies won or secrets unlocked. Some players want to
+*compete* in it. This could be hitting high scores or accomplishing
+speedruns. These two ideas map to the concepts of *achievements* and
+*leaderboards*.
+
+![A simple graphic representing the four types of motivation explained above](/assets/images/docs/cookbook/types-of-gamer-motivations.png)
+
+Ecosystems such as the App Store and Google Play provide centralized
+services for achievements and leaderboards. Players have achievements
+from all the games in one place and developers don't need to
+re-implement them for every game.
+
+This recipe demonstrates how to use the `games_services` package to add
+achievements and leaderboard functionality to your mobile game.
+
+## 1. Enable platform services
+
+To enable games services, set up *Game Center* on iOS and *Google Play
+Games Services* on Android.
+
+### iOS
+
+To enable Game Center (GameKit) on iOS:
+
+1.  Open your Flutter project in Xcode.
+    Open `ios/Runner.xcworkspace`
+
+2.  Select the root **Runner** project.
+
+3.  Go to the **Signing & Capabilities** tab.
+
+4.  Click the `+` button to add **Game Center** as a capability.
+
+5.  Close Xcode.
+
+6.  If you haven't already, register your game 
```

Comment:
I'm not super familiar with iOS development or deployment, but does this require being part of the Apple developer program?

If so, can we add a mention here or the the beginning of the iOS section that this a prerequisite and link to the relevant page. I believe we have some examples of this on the site already.

Reply:
I feel that might be going too far? Not even the page you're mentioning (https://docs.flutter.dev/deployment/ios) talks about the Apple developer program sign up process. As much as I like to inline knowledge so people don't need to go hunting for it elsewhere, I think that teaching people here that, for an iOS game, they need an Apple Developer account, is excessive.

That said, I don't _mind_ putting it here. In the end, it's a judgement call. I'd lean towards not mentioning such things, that's all.

Reply:
That makes sense. I'm fine leaving the note out. Thanks!